package.json

```diff
@@ -69,9 +69,10 @@
   },
   "dependencies": {
     "@decentralized-identity/ion-tools": "1.0.6",
-    "@tbd54566975/dwn-sdk-js": "0.0.26",
+    "@tbd54566975/dwn-sdk-js": "0.0.30-unstable-2023-04-15-149c713",
```

Comment:
@frankhinek i just published `0.0.30`. should be good to go on your end if you want to remove `-unstable-2023-04-15-149c713`

Reply:
You can ignore this in `simple-agent` Mic -- it was only for testing.  Thanks @mistermoe -- will update to `0.0.30`

Reply:
Although it shouldn't have presented any issues, I tested again with `0.0.30` because it did include one additional commit  beyond `0.0.30-unstable-2023-04-15-149c713`.  No issues, as expected, so bumped to 0.0.30.